docs/Formula-Cookbook.md

```diff
@@ -245,7 +245,7 @@ brew install --verbose --debug foo
 
 `--debug` will ask you to open an interactive shell if the build fails so you can try to figure out what went wrong.
 
-Check the top of the e.g. `./configure` output. Some configure scripts do not recognize e.g. `--disable-debug`. If you see a warning about it, remove the option from the formula.
+Check the top of the e.g. `./configure` output. Some configure scripts do not recognise e.g. `--disable-debug`. If you see a warning about it, remove the option from the formula.
```

Comment:
🇺🇸 et al. vs 🇬🇧 et al. What does the project prefer?

Reply:
The [style guide](https://docs.brew.sh/Prose-Style-Guidelines) prefers Commonwealth English, as do [previous commits](https://github.com/Homebrew/brew/commit/c1376f9f18f6e5da3d2290d5cdd808d32767068b#diff-e3e2a9bfd88566b05001b02a3f51d286).